package.json

```diff
@@ -71,10 +71,12 @@
     "delay-cli": "^1.1.0",
     "faucet": "0.0.1",
     "linkinator": "^1.6.0",
-    "markdownlint-cli": "^0.18.0",
     "nock": "^11.4.0",
     "pre-commit": "^1.2.2",
     "proxyquire": "^2.1.3",
+    "remark-cli": "^7.0.0",
+    "remark-frontmatter": "^1.3.2",
+    "remark-preset-lint-node": "^1.10.1-0",
```

Comment:
Are we good to land this once we bump to 1.10.1?

Reply:
I think so, I've re-installed and rebased